### PR TITLE
Suggest --list-subs when no matching subtitle format found (Resolve #9760)

### LINKED ISSUE #9760: Suggest `--list-subs` when no matching subtitle format found
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that I will be **blocked** if I *intentionally* remove or skip any mandatory\* field

### Checklist

- [X] I'm requesting a feature unrelated to a specific site
- [X] I've looked through the [README](https://github.com/yt-dlp/yt-dlp#readme)
- [X] I've verified that I have **updated yt-dlp to nightly or master** ([update instructions](https://github.com/yt-dlp/yt-dlp#update-channels))
- [X] I've searched [known issues](https://github.com/yt-dlp/yt-dlp/issues/3766) and the [bugtracker](https://github.com/yt-dlp/yt-dlp/issues?q=) for similar issues **including closed ones**. DO NOT post duplicates
- [X] I've read the [guidelines for opening an issue](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#opening-an-issue)

### Provide a description that is worded well enough to be understood

When you specify a subtitle format and it happens to be unavailable, the following is printed to stderr:

```
WARNING: No subtitle format found matching "srt" for language en-US, using ass
```
It would be nice if the warning could be updated to mention `--list-subs` for subtitle format errors.

### Provide verbose output that clearly demonstrates the problem

- [ ] Run **your** yt-dlp command with **-vU** flag added (`yt-dlp -vU <your command line>`)
- [ ] If using API, add `'verbose': True` to `YoutubeDL` params instead
- [ ] Copy the WHOLE output (starting with `[debug] Command-line config`) and insert it below

### Complete Verbose Output

```shell
N/A
```

### PR DESCRIPTION
**Pull request by DaPotato69**
**IMPORTANT**: PRs without the template will be CLOSED

### Description of your *pull request* and other information

Added "add the --list-subs flag to see a list of available subtitle formats" to YoutubeDL.py. 

also its my first issue :)

Fixes #9760 

<details open><summary>Template</summary> 

### Before submitting a *pull request* make sure you have:
- [x] At least skimmed through [contributing guidelines](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions) including [yt-dlp coding conventions](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#yt-dlp-coding-conventions)
- [x] [Searched](https://github.com/yt-dlp/yt-dlp/search?q=is%3Apr&type=Issues) the bugtracker for similar pull requests
- [x] Checked the code with [flake8](https://pypi.python.org/pypi/flake8) and [ran relevant tests](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#developer-instructions)

### In order to be accepted and merged into yt-dlp each piece of code must be in public domain or released under [Unlicense](http://unlicense.org/). Check all of the following options that apply:
- [x] I am the original author of this code and I am willing to release it under [Unlicense](http://unlicense.org/)
- [ ] I am not the original author of this code but it is in public domain or released under [Unlicense](http://unlicense.org/) (provide reliable evidence)

### What is the purpose of your *pull request*?
- [ ] Fix or improvement to an extractor (Make sure to add/update tests)
- [ ] New extractor ([Piracy websites will not be accepted](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#is-the-website-primarily-used-for-piracy))
- [x] Core bug fix/improvement
- [ ] New feature (It is strongly [recommended to open an issue first](https://github.com/yt-dlp/yt-dlp/blob/master/CONTRIBUTING.md#adding-new-feature-or-making-overarching-changes))

</details>

### CODE REVIEW

File: yt_dlp/YoutubeDL.py [hunk 1]
```diff
@@ -3071,7 +3071,7 @@ def process_subtitles(self, video_id, normal_subtitles, automatic_captions):
                 f = formats[-1]
                 self.report_warning(
                     'No subtitle format found matching "%s" for language %s, '
-                    'using %s' % (formats_query, lang, f['ext']))
+                    'using %s. Add the --list-subs flag to see a list of available subtitle formats' % (formats_query, lang, f['ext']))
```
Comment (pukkandan) [inline yt_dlp/YoutubeDL.py:3074]:
```suggestion
                    'using %s. Use --list-subs for a list of available subtitles' % (formats_query, lang, f['ext']))
```
to mirror the language used in `--format`'s hint

**Review by pukkandan — APPROVED** [review]

**Review by bashonly — APPROVED** [review]
LGTM. In the future, please do not resolve review comments unless you are directly applying the suggested change(s)

**Comment by DaPotato69** [comment]
so what do I do now?

**Comment by seproDev** [comment]
Nothing to do from your side. At some point a maintainer will merge this.

**Pull request merged**